parser.go

```diff
@@ -0,0 +1,137 @@
+package voyager
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var noTxPrefix = regexp.MustCompile("^\\s*--\\s+(NO_TRANSACTION)")
+var migrationDirection = regexp.MustCompile("\\.(up|down)\\.")
+var goMigrationFuncName = regexp.MustCompile("(Up|Down)_[0-9]*")
```

Comment:
Once more for consistency...

```suggestion
var goMigrationFuncName = regexp.MustCompile(`(Up|Down)_[0-9]*`)
```